src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,70 @@
+import { SortType } from '../../types/SortType';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  sortField: SortType;
+  setSortField: (field: SortType) => void;
+  todos: Todo[];
+  clearCompleted: () => void;
+};
+
+export const Footer: React.FC<Props> = ({
+  sortField,
+  setSortField,
+  todos,
+  clearCompleted,
+}) => {
+  const hasCompletedTodo = todos.some(todo => todo.completed);
+
+  const activeTodos = todos.reduce(
+    (acc, todo) => (todo.completed ? acc : acc + 1),
+    0,
+  );
+
+  return (
+    <footer className="todoapp__footer" data-cy="Footer">
+      <span className="todo-count" data-cy="TodosCounter">
+        {activeTodos} items left
+      </span>
+
+      <nav className="filter" data-cy="Filter">
+        <a
+          href="#/"
+          className={`filter__link ${sortField === SortType.All && 'selected'}`}
+          data-cy="FilterLinkAll"
+          onClick={() => setSortField(SortType.All)}
+        >
+          All
+        </a>
```

Comment:
As you see, <a> elements are almost the same, so you can 

1. Create separate component , say `FooterLink` (optional)
2. Iterate through `SortType` keys and render items for each key (please do it)

Reply:
thanks for that! I will fix it

Reply:
![image](https://github.com/mate-academy/react_todo-app-with-api/assets/140169867/70d08f6a-68e3-4fc9-b321-db063c122c20)
![image](https://github.com/mate-academy/react_todo-app-with-api/assets/140169867/6c58467c-3c8d-4202-b849-a4b1cffa7035)
![image](https://github.com/mate-academy/react_todo-app-with-api/assets/140169867/e84e6cc3-2a5e-4a02-bf28-fd11c0b4db30)
![image](https://github.com/mate-academy/react_todo-app-with-api/assets/140169867/fc3eebfd-12f4-40b4-b024-b802db87551c)
![image](https://github.com/mate-academy/react_todo-app-with-api/assets/140169867/e7647f3e-ccb3-4e96-bd96-9d265147bf8a)
Hello! I tried implement that, but on working demo on All button should be just '#/' address. Therefore, I left buttons unchanged